common/dynamicconfig/configstore/config_store_client.go

```diff
@@ -334,11 +352,6 @@ func (csc *configStoreClient) updateValue(name dc.Key, value interface{}, retryA
 			}
 		}
 	} else {
-		dcValues, ok := value.([]*types.DynamicConfigValue)
-		if !ok {
-			return errors.New("invalid value")
-		}
-
```

Comment:
ah, they moved from here.  great, looks correct then, and 👍👍 to clearer type safety.